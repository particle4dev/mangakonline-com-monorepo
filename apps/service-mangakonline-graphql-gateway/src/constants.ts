import * as path from 'path';

const { NODE_ENV = 'development' } = process.env;

export const basePath = NODE_ENV === 'production' ? process.cwd() : path.resolve(process.cwd(), 'apps', 'service-mangakonline-graphql-gateway');

export const envFilePath = [path.join(basePath, 'env', '.env')];

if(NODE_ENV === 'development') {
  envFilePath.push(path.join(basePath, 'env', '.env-development'));
}

if(NODE_ENV === 'test') {
  envFilePath.push(path.join(basePath, 'env', '.env-test'));
}

if(NODE_ENV === 'production') {
  envFilePath.push(path.join(basePath, 'env', '.env-production'));
}
