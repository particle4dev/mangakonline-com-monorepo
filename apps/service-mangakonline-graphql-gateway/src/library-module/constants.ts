export const MODULE_CONNECTION_NAME = 'library_Module_Connection';
