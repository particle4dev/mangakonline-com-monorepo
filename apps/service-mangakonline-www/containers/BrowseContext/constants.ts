const createSymbol = (name: string) => `components/BrowseContext/${name}`;

export const UPDATE_PATH = createSymbol('UPDATE_PATH');
export const INITIALIZING = createSymbol('INITIALIZING');
export const INITIALIZED = createSymbol('INITIALIZED');
