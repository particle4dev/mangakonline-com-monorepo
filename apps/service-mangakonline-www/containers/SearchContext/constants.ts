const createSymbol = (name: string) => `containers/SearchContext/${name}`;

export const UPDATE_PATH = createSymbol('UPDATE_PATH');

export const INITIALIZING = createSymbol('INITIALIZING');

export const INITIALIZED = createSymbol('INITIALIZED');
