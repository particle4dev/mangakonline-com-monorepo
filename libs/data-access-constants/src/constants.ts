// jwt token cookie name
export const JWT_ACCESS_TOKEN_KEY = "JWT_ACCESS_TOKEN_KEY";
// jwt algorithm
export const JWT_ACCESS_TOKEN_ALGORITHM = "HS256";
// how long token will be expired
export const JWT_ACCESS_TOKEN_EXPIRES_IN = "15m";

export const JWT_REFRESH_TOKEN_KEY = "JWT_REFRESH_TOKEN_KEY";

export const JWT_REFRESH_TOKEN_ALGORITHM = "HS256";

export const JWT_REFRESH_TOKEN_EXPIRES_IN = "7d";

export const JWT_RESET_PASSWORD_TOKEN_ALGORITHM = "HS256";

export const JWT_RESET_PASSWORD_TOKEN_EXPIRES_IN = "1d";

export const JWT_EMAIL_TOKEN_ALGORITHM = "HS256";

export const JWT_EMAIL_TOKEN_EXPIRES_IN = "1d";

export const AUTH_HEADER = "authorization";
