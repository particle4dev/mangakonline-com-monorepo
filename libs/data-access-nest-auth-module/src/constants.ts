export const MODULE_CONNECTION_NAME = 'User_Module_Connection';
