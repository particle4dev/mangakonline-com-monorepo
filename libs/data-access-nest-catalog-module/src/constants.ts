export const MODULE_CONNECTION_NAME = 'Catalog_Module_Connection2';
