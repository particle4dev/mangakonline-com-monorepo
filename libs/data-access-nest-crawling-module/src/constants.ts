export const MODULE_CONNECTION_NAME = 'Crawling_Module_Connection';

export enum TASK_STATUS_TYPE {
  NEW = 'new',
  IN_PROGRESS = 'in_progress',
  COMPLETE = 'complete',
  RETRY_REQUIRED = 'retry_required',
  ERROR = 'error',
  TO_BE_DELETED = 'to_be_deleted'
};
