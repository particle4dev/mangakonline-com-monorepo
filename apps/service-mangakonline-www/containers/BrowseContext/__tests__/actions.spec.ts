import {
  updatePath,
} from '../actions';
import { UPDATE_PATH } from '../constants';

describe('containers/BrowseContext/actions/updatePath', () => {
  const params = {
    pathname: '/path',
    query: {}
  };
  it('should updatePath should create updatePath action', () => {
    expect(updatePath(params)).toMatchSnapshot();
  });

  it('should return the correct type and the passed name', () => {
    const expectedResult = {
      type: UPDATE_PATH,
      payload: {
        params
      }
    };

    expect(updatePath(params)).toEqual(expectedResult);
  });
});
