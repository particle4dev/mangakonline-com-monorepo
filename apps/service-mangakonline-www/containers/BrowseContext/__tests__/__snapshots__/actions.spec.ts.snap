// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`containers/BrowseContext/actions/updatePath should updatePath should create updatePath action 1`] = `
Object {
  "payload": Object {
    "params": Object {
      "pathname": "/path",
      "query": Object {},
    },
  },
  "type": "components/BrowseContext/UPDATE_PATH",
}
`;
